ui/src/pages/playerStats/PlayerStats.tsx

```diff
@@ -18,7 +18,9 @@ import {
   IUser,
   IAllStats,
 } from "../../services/apiTypes";
-import EloChart, {ChartData} from "../../components/eloChart/eloChart";
+import EloChart, { ChartData } from "../../components/eloChart/eloChart";
+
+const STARTING_ELO: number = 1000;
```

Comment:
should perhaps have this in a constants file somewhere, presumably the same number is used when registering a new user

Reply:
yes it is - the backend is agnostic to the starting value i believe